src/current/_includes/metrics-usage/tenant.consumption.pgwire_egress_bytes.md

```diff
@@ -0,0 +1,5 @@
+The number of RUs that were consumed by byte traffic to the client. Egress bytes are converted to Request Units using this equivalency:
+<br>
+<b>1 RU = 1 KiB Network egress.</b>
+<br>
+Correlate this metric with Request Units (RUs). <a href="https://www.cockroachlabs.com/docs/cockroachcloud/serverless-resource-usage">Learn more</a>.
```

Comment:
same comment here and everywhere about links called 'learn more' for reasons given in previous comment.  suggestion for rewrite can follow same basic pattern as given in that comment